deis-logger/manifests/deis-logger-rc.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: v1
 kind: ReplicationController
 metadata:
   name: deis-logger
-  namespace: deis
```

Comment:
yes. sorry bout that I took the manifest from the logger repo which for some reason doesnt have the namespace. Added it to this PR